lib/services/local/presence.go

```diff
@@ -202,6 +202,22 @@ func (s *PresenceService) DeleteNode(ctx context.Context, namespace string, name
 	return s.Delete(ctx, key)
 }
 
+// GetNode returns a node by name and namespace.
+func (s *PresenceService) GetNode(ctx context.Context, namespace, name string) (types.Server, error) {
+	if namespace == "" {
+		return nil, trace.BadParameter("missing parameter namespace")
+	}
+	if name == "" {
+		return nil, trace.BadParameter("missing parameter name")
+	}
+	item, err := s.Get(ctx, backend.Key(nodesPrefix, namespace, name))
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	return services.UnmarshalServer(item.Value, types.KindNode, services.SkipValidation(),
+		services.WithResourceID(item.ID), services.WithExpires(item.Expires))
```

Comment:
Why do we force `SkipValidation` here? Shouldn't this function, like the other functions in this file, accept `opts ...services.MarshalOption` allowing the caller to pass in `SkipValidation` when needed?

Reply:
IIUC `SkipValidation` is fine here because it's taking pre validated data from the server. Some other getters do this, such as `GetToken`, `GetAppSession`. 

Also more to your question, it'd be an undue challenge to add the `MarshalOptions` parameter due to the way we use the same `ClientI` interface for both the client and internal server functions. The gRPC client doesn't need to / can't use `MarshalOptions` so this wouldn't be possible without some big refactors. Eventually we need to refactor these interfaces to make more sense and avoid these issues.